site/docs/master/oracle-config.md

```diff
@@ -0,0 +1,244 @@
+
+   ```
+   kubectl get deployments --namespace=nginx-example
+   kubectl get services --namespace=nginx-example
+   kubectl get namespace/nginx-example
+   ```
+
+   This should return: `No resources found.`
+
+4. Restore your lost resources: `velero restore create --from-backup nginx-backup`
+
+   ```
+   $ velero restore create --from-backup nginx-backup
+   Restore request "nginx-backup-20190604102710" submitted successfully.
+   Run `velero restore describe nginx-backup-20190604102710` or `velero restore logs nginx-backup-20190604102710` for more details.
+   ```
+
+   Running `kubectl get namespaces` will show that the `nginx-example` namespace has been restored along with its contents. 
+
+5. Run: `velero restore get` to view the list of restored resources. After the restore finishes, the output looks like the following:
+
+   ```
+   $ velero restore get
+   NAME                          BACKUP         STATUS      WARNINGS   ERRORS   CREATED                         SELECTOR
+   nginx-backup-20190604104249   nginx-backup   Completed   0          0        2019-06-04 10:42:39 -0700 PDT   <none>
+   ```
+
+   NOTE: The restore can take a few moments to finish. During this time, the `STATUS` column reads `InProgress`. 
+
+   After a successful restore, the `STATUS` column shows `Completed`, and `WARNINGS` and `ERRORS` will show `0`. All objects in the `nginx-example` namespace should be just as they were before you deleted them.
+
+   If there are errors or warnings, for instance if the `STATUS` column displays `FAILED` instead of `InProgress`, you can look at them in detail with `velero restore describe <RESTORE_NAME>`
+
+
+6. Clean up the environment with `kubectl delete -f examples/nginx-app/base.yaml` 
+
+   ```
+   $ kubectl delete -f examples/nginx-app/base.yaml
+   namespace "nginx-example" deleted
+   deployment.apps "nginx-deployment" deleted
+   service "my-nginx" deleted
+   ```
+
+   If you want to delete any backups you created, including data in object storage, you can run: `velero backup delete BACKUP_NAME`
+
+   ```
+   $ velero backup delete nginx-backup
+   Are you sure you want to continue (Y/N)? Y
+   Request to delete backup "nginx-backup" submitted successfully.
+   The backup will be fully deleted after all associated data (disk snapshots, backup files, restores) are removed.
+   ```
+
+   This asks the Velero server to delete all backup data associated with `BACKUP_NAME`. You need to do this for each backup you want to permanently delete. A future version of Velero will allow you to delete multiple backups by name or label selector.
+
+   Once fully removed, the backup is no longer visible when you run: `velero backup get BACKUP_NAME` or more generally `velero backup get`:
+   
+   ```
+   $ velero backup get nginx-backup
+   An error occurred: backups.velero.io "nginx-backup" not found
+
+   $ velero backup get 
+   NAME     STATUS      CREATED     EXPIRES     STORAGE     LOCATION        SELECTOR
+   
+    ```
```

Comment:
Looks like maybe this has one leading space too many, it's not being rendered properly - see https://deploy-preview-1570--velero.netlify.com/docs/master/oracle-config/

Reply:
Just to clarify, the line with the 3 backticks to close the code block (L236 in the most recent version) is the one with one too many leading spaces.

Reply:
Thanks for following up! I just caught it and I'll update one more time.

Reply:
Fingers crossed 🤞

Reply:
woohoo looks good on the preview!